diabetes_regression/util/model_helper.py

```diff
@@ -40,25 +40,40 @@ def get_model(
     (optional) tag (str): the tag value & name the model was registered under.
 
     Return:
-    A single aml model from the workspace that matches the name and tag.
+    A single aml model from the workspace that matches the name and tag, or
+    None.
     """
     if aml_workspace is None:
         print("No workspace defined - using current experiment workspace.")
         aml_workspace = get_current_workspace()
 
-    if tag_name is not None and tag_value is not None:
+    tags = None
+    if tag_name is not None or tag_value is not None:
+        # Both a name and value must be specified to use tags.
+        if tag_name is None or tag_value is None:
+            raise ValueError(
+                "model_tag_name and model_tag_value should both be supplied"
+                + "or excluded"  # NOQA: E501
+            )
+        tags = [[tag_name, tag_value]]
+
+    model = None
+    if model_version is not None:
+        # TODO(tcare): Finding a specific version currently expects exceptions
+        # to propagate in the case we can't find the model. This call may
+        # result in a WebserviceException that may or may not be due to the
+        # model not existing.
```

Comment:
Deployment isn't using this method - why WebserviceException? It should be just the training and batch scoring pipeline that uses this.

Reply:
Nevermind - I see what you're saying. Can you make a new issue for this?

Reply:
Will do :)